Gopkg.toml

```diff
@@ -1,23 +1,31 @@
 [[constraint]]
-  branch = "master"
   name = "github.com/container-storage-interface/spec"
+  version = "~0.2.0"
```

Comment:
~0.2.0?

Reply:
It is done the same way across all CSI external components (registart, attacher, provisioner, etc)